tools/syncsnoop

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python
```

Comment:
I spoke too soon! I should have looked at some actual examples, since I found that most (all?) of my installed machine uses /usr/bin/python explicitly. Sorry for flip-flopping, but since I already have the other comment pending, can you change this back? Shame on me :)